main.go

```diff
@@ -30,6 +30,7 @@ const (
 	ENV_WG_TUN_FD             = "WG_TUN_FD"
 	ENV_WG_UAPI_FD            = "WG_UAPI_FD"
 	ENV_WG_PROCESS_FOREGROUND = "WG_PROCESS_FOREGROUND"
+	ENV_WG_DISABLE_OFFLOAD    = "WG_DISABLE_OFFLOAD"
```

Comment:
Let's not add a nob. If it's broken, we'll fix what's broken, or revert if we can't fix it, both of which are better than circulating magic flags to sometimes fix issues people have.

Reply:
Ok.